.github/workflows/test-cdk-cd.yml

```diff
@@ -0,0 +1,51 @@
+name: Test tiles generation CDK CD and ECS task run
+
+on: 
+  pull_request:
+      branches:
+        - main
+      paths:
+        - 'tiles-generation/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      
+      - name: Use Node.js
+        uses: actions/setup-node@v3.0.0
+        with:
+          node-version: '16.x'
+      
+      - name: Install cdk-cli & cdk-lib
+        run: |
+          npm install -g aws-cdk 
+          cd tiles-generation/cdk
+          npm install aws-cdk-lib
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
```

Comment:
same comment as Vijay's, we can do it configurable

Reply:
sure, will update to make it configurable.